source/initial-dns-seedlist-discovery/initial-dns-seedlist-discovery.rst

```diff
@@ -89,16 +89,20 @@ Default Connection String Options
 
 A Client MUST also query the DNS server for TXT records. If available, these
 TXT records provide default connection string options. In most cases only one
-TXT record is necessary, unless the default options exceed the maximum length
-of 255 characters that TXT records allow for. To allow for this case, a Client
-MUST allow for multiple TXT records for the same host name and consider the
-options in all of them.
+TXT record is necessary. The maximum length of a TXT record string is 255
+characters, but there can be multiple strings per TXT record. A Client MUST
+support multiple TXT record strings and concatenate them as they were one
+single string in the order they are define in in the TXT record. The maximum
```

Comment:
"they are defined in each TXT record"

But is the order *among* the TXT records guaranteed or not? A couple paragraphs down you say there's no guaranteed order.